### PR TITLE
Mobile UCR: option to hide table

### PR DESCRIPTION
**Pull request by orangejenny**
http://manage.dimagi.com/default.asp?249206

screen shot 2017-04-06 at 12 46 15 pm

`w=1`

@esoergel / @nickpell

### CODE REVIEW

File: corehq/apps/app_manager/suite_xml/features/mobile_ucr.py [hunk 1]
```diff
@@ -154,59 +154,62 @@ def _get_description_text(report_config):
                 locale=Locale(id=id_strings.report_description(report_config.uuid))
             )
 
-    return models.Detail(custom_xml=Detail(
-        id='reports.{}.summary'.format(config.uuid),
+    detail_id='reports.{}.summary'.format(config.uuid)
```
Comment (dimagimon) [inline corehq/apps/app_manager/suite_xml/features/mobile_ucr.py]: E225 missing whitespace around operator

Comment (dimagimon) [inline corehq/apps/app_manager/suite_xml/features/mobile_ucr.py]: E225 missing whitespace around operator

**Comment by nickpell** [comment]
Looks good.  Is it reasonable to add a check to https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/test_suite.py#L701 for this use case?

**Comment by orangejenny** [comment]
@nickpell yep, thanks for the prod. test added.

**Pull request merged**